packages/suite/src/components/suite/modals/AddAccount/components/AccountTypeDescription.tsx

```diff
@@ -50,9 +50,9 @@ export const AccountTypeDescription = ({ network, accountTypes }: Props) => {
     return (
         <Info size="small" textAlign="left">
             <Translation id={description} />{' '}
-            <ExternalLink href={url} size="small">
+            <TrezorLink href={url} size="small">
```

Comment:
Icon is missing in your implementation

![Screenshot 2021-09-07 at 14 31 37](https://user-images.githubusercontent.com/33235762/132344981-36751083-aa34-4ba9-a3b5-0a2e4c600646.png)
![Screenshot 2021-09-07 at 14 31 41](https://user-images.githubusercontent.com/33235762/132344986-4cbdf3d0-e61e-4602-95f4-19a844e61b7a.png)